submissions/Eugene-CG/DOM/index.js

```diff
@@ -0,0 +1,136 @@
+const texts = [
+  {
+    identifier: "HahahaOne",
+    text: `Lorem One one One one One one One one One One one One one One one One one One One one One one One
+    one One one One One one One one One one One one Oneoneipsum dolor 
+    sit amet consectetur adipisicing elit. 
+    Ad, qui
+    laboriosam? Est, et ex? Fugit, 
+    quidem aliquid cumque porro rerum
+    maiores voluptate cupiditate deserunt tenetur distinctio mollitia
+    inventore quo! Nostrum.`,
+  },
+  {
+    identifier: "HahahaTwo",
+    text: `Two Two Two Two Two Two Two Two Two Two Two
+    ipsum dolor sit amet consectetur adipisicing elit. Ad, qui
+    laboriosam? Est, et ex? Fugit, 
+    quidem aliquid cumque porro rerum
+    maiores voluptate cupiditate deserunt tenetur distinctio mollitia
+    inventore quo! Nostrum.`,
+  },
+  {
+    identifier: "HahahaThree",
+    text: `Lorem Three Three Three Three Three Three Three Three Three Three Thr ipsum dolor sit amet consectetur adipisicing elit. Ad, qui
+    laboriosam? Est, et ex? Fugit, quidem aliquid
+     cumque porro rerum
+    maiores voluptate cupiditate deserunt tenetur distinctio mollitia
+    inventore quo! Nostrum.`,
+  },
+  {
+    identifier: "HahahaFour",
+    text: `Lorem Four Four Four Four Four Four Four Four Four Four Four Four Fo
+    ipsum dolor sit amet consectetur adipisicing elit. Ad, qui
+    laboriosam? Est, et ex? Fugit, quidem aliquid cumque porro rerum
+    maiores voluptate cupiditate deserunt tenetur distinctio mollitia
+    inventore quo! Nostrum.`,
+  },
+  {
+    identifier: "HahahaFive",
+    text: `Lorem Five Five Five Five Five Five Five Five Five Five Five Five Fi
+    ipsum dolor sit amet consectetur adipisicing elit. Ad, qui
+    laboriosam? Est, et ex? Fugit, quidem aliquid cumque porro rerum
+    maiores voluptate cupiditate deserunt tenetur distinctio mollitia
+    inventore quo! Nostrum.`,
+  },
+  {
+    identifier: "Something",
+    text: `Lorem Something Something ipsum dolor sit amet consectetur adipisicing elit. Ad, qui
+    laboriosam? Est, et ex? Fugit, quidem aliquid cumque porro rerum
+    maiores voluptate cupiditate deserunt tenetur distinctio mollitia
+    inventore quo! Nostrum.`,
+  },
+];
+const wrapper = document.querySelector(".wrapper");
+const mainContent = document.querySelector(".main__content");
+const menuList = document.querySelector(".menu__list");
+
+const fillMenuList = (event) => {
+  texts.reduceRight((accumulatorObj, currObj) => {
+    const li = document.createElement("LI");
+    menuList.appendChild(li);
+
+    const a = document.createElement("A");
+    a.setAttribute("href", "#");
+    a.classList.add("menu__link");
+    a.innerText = currObj.identifier;
+    li.appendChild(a);
+  }, "Hello Alexandr :D");
+};
+
+const delegateEvents = (event) => {
+  const { target } = event;
```

Comment:
When the user clicks on the menu item to URL added `#` sign, avoid this behaviour 
![image](https://user-images.githubusercontent.com/16196199/183917102-f8d2efdd-b2af-4aca-9e0f-8e9a1bc7d2f2.png)

Reply:
Added preventDefault to event in handleList function